[ET-VK] Enable Vulkan tests in CI

SS-JIA opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #3737

## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

Differential Revision: [D57968974](https://our.internmc.facebook.com/intern/diff/D57968974)

SS-JIA committed:
[ET-VK] Enable Vulkan tests in CI

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

[ghstack-poisoned]

kirklandsign commented on .ci/scripts/setup-vulkan-linux-deps.sh:
```diff
@@ -0,0 +1,48 @@
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -ex
```
BTW is this script used for developers (non CI) as well?

SS-JIA replied:
nope, just for CI

SS-JIA commented:
@SS-JIA has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D57968974).

SS-JIA committed:
Update on "[ET-VK] Enable Vulkan tests in CI"


## Context

As title; enables Vulkan delegate to build and test during the unit test workflow.

Differential Revision: [D57968974](https://our.internmc.facebook.com/intern/diff/D57968974)

[ghstack-poisoned]

SS-JIA commented:
@SS-JIA has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D57968974).
